packages/components/src/utils/colors.js

```diff
@@ -2,7 +2,7 @@
  * External dependencies
  */
 import { get } from 'lodash';
-import hexRgb from 'hex-rgb';
+import { parseToRgb } from 'polished';
```

Comment:
I think we already have tinycolor2 in our bundles. We could use it instead right?

Reply:
@jameslnewell Ah, you folks are so fast! You jumped on this before I could :)
I was going to suggest using `tinycolor2` since it was already there. My apologies for not realizing that when I submitted my original PR.

Thanks for the heads up @youknowriad + @aduth !